project/sites/sentencereview/themes/sentencereview_theme/src/scss/components.scss

```diff
@@ -93,7 +93,8 @@
   font-weight: bold;
 }
 
-.front {
+.front ,
+.node-type-basic-page {
   .content-sidebar-wrap {
     width: 65%;
```

Comment:
I think if you change this variable https://github.com/dof-dss/Unity_svr_2/blob/6662708a6297553b1695b62678d0af752edf47a9/project/sites/sentencereview/themes/sentencereview_theme/src/scss/init/_variables.scss#L50 to $two-col-layout-sidebar-second: (65%, 30%); and add your sidebar menu in it might fix it without this?
If not you only want it to be 65% at big screen and 100% on smaller screens so something like this might be better
`@include breakpoint($tab) {
    width: 65%;
}`